integration_test/Bindings/Tcl/Inputs/simple.mlir

```diff
@@ -1,23 +1,23 @@
-hw.module.extern @ichi(%a: i2, %b: i3) -> (c: i4, d: i5)
+hw.module.extern @ichi(input %a: i2, input %b: i3, output %c: i4, d: i5)
 
-hw.module @owo() -> (owo_result : i32) {
+hw.module @owo(, output %owo_result : i32) {
```

Comment:
Are you planning on doing away with the initial comma here?

Reply:
Not everything passes yet, hence the draft status.  Lots of mechanical stuff...